.gitignore

```diff
@@ -14,6 +14,9 @@
 #################################
 /nbproject
 .idea
+/.project
+/.buildpath
+/.settings/
```

Comment:
For now just dont add newlines. I think that's all that is necessary at this point.
Keep your PR/Changes as isolated and noise clean as possible always.